<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>issueScreen</class>
 <widget class="QMainWindow" name="issueScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>470</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>LMS Workspace</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="empPinLbl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Employee PIN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="book1Lbl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Book 1 Code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="book2Lbl">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Book 2 Code:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="book2Text">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="empPinText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="book1Text">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="issueBookBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Issue Books</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="bookEditText">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEmpText">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bookEditLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter Book Code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="editEmpLbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter Employee PIN:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editBookDetails">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Book Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editEmpDetails">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Employee Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generateReportBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Reports</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBooksBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Books</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
